cloud-hosted/before-start-information.md

```diff
@@ -0,0 +1,7 @@
+# Instructions for this Cloud Hosted Guide
```

Comment:
No capital for "Cloud Hosted Guide"
-> "cloud-hosted guide"

Reply:
Done